cmd/debos/debos.go

```diff
@@ -28,6 +28,7 @@ func checkError(context *debos.DebosContext, err error, a debos.Action, stage st
 
 func do_run(r actions.Recipe, context *debos.DebosContext) int {
 	for _, a := range r.Actions {
+		log.Printf("==== %s ====\n", a)
```

Comment:
does this also print when running nest recipes with the `recipe` action? :p

Reply:
Oops fixed, or at least `tests/recipes/test.yaml` seems to be happy.